frontend/src/components/Cases/AccordionItem.js

```diff
@@ -0,0 +1,199 @@
+import AccordionSvg from "./AccordionSvg";
+import { schemaDisplay, copyToClipboard } from "../../utilities/utilFunctions";
+import Svg from "../../assets/svg/Svg";
+
+const AccordionItem = ({ eachCase, implementations, caseImplementation }) => {
+  const seq = eachCase.seq;
+  const description = eachCase.case.description;
+  const schema = eachCase.case.schema;
+  const tests = eachCase.case.tests;
+
+  function result(index, implementation) {
+    var testResult;
+    if (implementation.skipped && implementation.skipped == true) {
+      return (testResult = "skipped");
+    } else if (implementation.caught && implementation.caught == true) {
+      return (testResult = "errored");
+    } else if (implementation.results && implementation.expected) {
+      var caseResults = implementation.results.filter(
+        (element) => element.skipped,
+      );
+      if (caseResults.length > 0) {
+        return (testResult = "skipped");
+      }
+      if (
+        implementation.results.every(
+          (each) => each.hasOwnProperty("errored") && typeof each === "object",
```

Comment:
If `each` indeed can be something different than `object`, the AND conditions should be reversed

Reply:
I think it unintentionally changed during some random try. Fixed it now!